src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {ln: { $ln: $int}}}]
+            cursor: {}
+
+- Name: ExpressionLog10
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionLog10
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {log10: { $log10: $int}}}]
+            cursor: {}
+
+- Name: ExpressionMod
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionMod
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {mod: { $mod: [$int, $int2]}}}]
+            cursor: {}
+
+- Name: ExpressionMultiply
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionMultiply
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {multiply: { $multiply: [$int, $int2]}}}]
+            cursor: {}
+
+- Name: ExpressionNot
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionNot
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {not: { $not: [{$gt: [$int, $int2]}]}}}]
+            cursor: {}
+
+- Name: ExpressionReplaceOne
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionReplaceOne
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {replaceOne: { $replaceOne: {input: $preDeterminedString, find: "one", replacement: "ZERO"}}}}]
```

Comment:
'predeterminedString' is misleading, isn't it random whether it contains "one" "two" or "three"? Anyways, I think we should put this to the test by always having the 'find' string within 'input'. Even better still, it would be good if the find string was at the very end of the input string

Reply:
Done.